src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -328,21 +326,6 @@ private void ThrowIfNotInitialized()
         }
     }
 
-    private static void EnsureDepsFile(Type entryPoint)
-    {
-        if (entryPoint.Assembly.EntryPoint == null)
-        {
-            throw new InvalidOperationException($"Assembly of specified type {entryPoint.Name} does not have an entry point.");
-        }
```

Comment:
This check may still be useful though...

Reply:
Added that back, and a corresponding test.